<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dictEdit</class>
 <widget class="QWidget" name="dictEdit">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>445</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>618</width>
    <height>445</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>edit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,4,1" columnstretch="1,1">
   <property name="topMargin">
    <number>20</number>
   </property>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="wordLo">
     <property name="spacing">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignBottom">
      <widget class="QLabel" name="WordLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Word:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QLineEdit" name="WordEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignBottom">
      <widget class="QLabel" name="TranslationLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Translation:</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QLineEdit" name="TranslationEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="editButtonLo" stretch="1,1,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="ItemBtn">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>New Item</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="AddBtn">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Save Item</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="delButton">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Remove Item</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="buttonLo">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="NewBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>195</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>New Dict</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="loadButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>196</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Load File</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignVCenter">
      <widget class="QPushButton" name="SaveBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>195</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Save File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="3">
    <layout class="QVBoxLayout" name="listLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>300</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
